### ISSUE TITLE
buildpack-update doesn't update nodejs-buildpack

### ISSUE THREAD

**Issue by qlibin** [springernature/cf-buildpack-update-action#2]
We have defined this buildpack-update action for one of our repositories: https://github.com/springernature/ecommerce-style-guides/blob/master/.github/workflows/buildpack-update.yml

The deployment manifest: https://github.com/springernature/ecommerce-style-guides/blob/master/ci/manifests/cf-live.yml

Build pack definition in the manifest: 
```yaml
  buildpacks:
    - https://github.com/cloudfoundry/nodejs-buildpack.git#v1.6.30
```

Even though the buildpack has newer versions available the action doesn't create PR.

Also no complaints in the action's log: https://github.com/springernature/ecommerce-style-guides/runs/5197704511?check_suite_focus=true#step:4:6

Available buildpack's version tags: https://github.com/cloudfoundry/nodejs-buildpack/tags

**Comment by jshiell** [comment]
This is because the manifest is called `cf-live.yml`, whereas we're filtering for the word `manifest`. I've added `*cf*` to the filter.

**Comment by jshiell** [comment]
Should be fixed in 1.0.3.

**Comment by efasel** [comment]
> Should be fixed in 1.0.3.

https://github.com/springernature/cf-buildpack-update-action/#keep-your-action-up-to-date :smile:
